plugins/techdocs-node/src/stages/generate/helpers.ts

```diff
@@ -141,6 +141,14 @@ export const MKDOCS_SCHEMA = DEFAULT_SCHEMA.extend([
     instanceOf: UnknownTag,
     construct: (data: string, type?: string) => new UnknownTag(data, type),
   }),
+  new Type('tag:', {
```

Comment:
do we need to define the tag name ?

Reply:
From my tests it's not strictly necessary but in the [js-yaml examples](https://github.com/nodeca/js-yaml/blob/master/examples/handle_unknown_types.js#L17) and in [this issue](https://github.com/nodeca/js-yaml/issues/636#issuecomment-1119714506) they seem to use it so I figured it would be better. Happy to remove it if you want.

Reply:
it can stay as is IMHO